pkg/agent/store_cred.go

```diff
@@ -178,16 +170,22 @@ func storeTerraformStateConfig(captenConfig config.CaptenConfig, agentClient age
 func storeNatsCredentials(captenConfig config.CaptenConfig, appGlobalVaules map[string]interface{}, vaultClient vaultcredpb.VaultCredClient) error {
 	val, err := randomTokenGeneration()
 	if err != nil {
-		return fmt.Errorf("Nats Token generation failed, %v", err)
+		return fmt.Errorf("nats Token generation failed, %v", err)
+	}
+
```

Comment:
it need to read all yaml files, based on yaml credential-type and other values, it need to create credential and update in capten